### PR TITLE
update docs with info about LTS 5.0

### PR DESCRIPTION
**Pull request by ru-fu**
Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: content/lxc/downloads.md [hunk 1]
```diff
@@ -14,16 +14,16 @@ For Ubuntu users, we have official PPAs for LXC:
 And for those who want development snapshots:
 
  * [lxc-git-master](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-master): "master" branch
- * [lxc-git-stable-3.0](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-stable-3.0): "stable-3.0" branch
  * [lxc-git-stable-4.0](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-stable-4.0): "stable-4.0" branch
+ * [lxc-git-stable-5.0](https://launchpad.net/~ubuntu-lxc/+archive/lxc-git-stable-5.0): "stable-5.0" branch
```
Comment (tomponline) [inline content/lxc/downloads.md:18]: @ru-fu @stgraber link doesn't exist.

**Review by tomponline — APPROVED** [review]
Apart from the broken link

File: content/lxc/downloads.md (hunk 1 above)
Comment (simondeziel) [inline content/lxc/downloads.md:18]: I'd put the `5.0` one where the `3.0` was to have a "chronological" order.

File: content/lxc/downloads.md [hunk 2]
```diff
@@ -14,16 +14,16 @@ For Ubuntu users, we have official PPAs for LXC:
...
 
 # Current development version
 
 LXC has three active git branches:
 
  * **master**: Current development branch
- * **stable-3.0**: Stable update branch for LXC 3.0.x
  * **stable-4.0**: Stable update branch for LXC 4.0.x
+ * **stable-5.0**: Stable update branch for LXC 5.0.x
```
Comment (simondeziel) [inline content/lxc/downloads.md:26]: Same here.

**Review by simondeziel — APPROVED** [review]

File: content/lxc/downloads.md (hunk 1 above)
Comment (ru-fu) [reply content/lxc/downloads.md:17]: Should it exist or should we remove the link?

Comment (tomponline) [reply content/lxc/downloads.md:17]: What do you think @stgraber ?

Comment (stgraber) [reply content/lxc/downloads.md:17]: It exists now (empty like the 4.0 one though as I don't have time to sort out those builds until we fully move to Github Actions).

Comment (tomponline) [reply content/lxc/downloads.md:17]: thanks!

**Pull request merged**